.eslintrc.js

```diff
@@ -1,28 +1,26 @@
 module.exports = {
-  parser: 'babel-eslint',
-  extends: ['standard', 'standard-react', 'prettier', 'react-app'],
-  plugins: ['prettier', 'react'],
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: ['plugin:react/recommended', 'airbnb', 'prettier'],
```

Comment:
I'd prefer using standard over airbnb tbh.

Reply:
But you wasn't. Standard is more simplified. I changed to airbnb and linter not claiming to much

Reply:
But yes - probably it's a good time to update this

Reply:
What do you mean I wasn't? It was right there in eslint before you replaced it with airbnb 🤔 
Airbnb's config has way too much silly rules IMO

Reply:
There is a good point https://github.com/BuildingXwithJS/BxJS-Website/pull/33#discussion_r398907002

Reply:
reverted